### DISCUSSION TITLE
0.7.0. Why does Autoortho create a *_bpk folder and put it at the top of the scenery_packs.ini?

### DISCUSSION THREAD

**Discussion by zippy77777** [kubilus1/autoortho#420] [General]
Everything I need to know is in the title
Can anyone explain why this is?
Debug set to False
Running with winFSP.
Thx

**Comment by Jeff2122** [comment]
You can safely delete those lines along with the folders by that name in your XP12\Custom Scenery folder. My experience with those folders is they appear when I go from v0.6.x (or v0.7.0) back to v0.5.1-3. I go back and forth between those and other versions all the time. Nowadays I delete any junction either of them make before starting the other. They each create the junctions they need to work when you press run. That is why you have to enter your path information on the configuration screen - so it knows where to create the folders. Create vs. use - if the junctions are there, the right term would be which folders to "use" I suppose. One version can't use the other version's folders. So it renames it .bkp for "backup" and creates new next to them.  
The exception to the above is that if you use the older version AND have your data on a separate drive - in that case, I must create junctions in my main XP folder to the junctions AO created on my remote data drive. Hope that makes sense.
I never have to touch my junctions or .ini after that, unless I swap version for testing purposes.

**Reply by zippy77777** [reply]
I delete the _bpk from the Custom Scenery folder but it just keeps coming back :(

**Comment by Jeff2122** [comment]
Do you use anything that plays with your .ini file? Two I know of are XOrganizer and SAM. I use neither so I'm not sure.

**Reply by zippy77777** [reply]
I don't use xOrganiser and SAM has gone

**Reply by zippy77777** [reply]
I lie I did delete it and it hasn't come back this time, thx mate

**Reply by Jeff2122** [reply]
ok you must have updated when I was typing. Good you got it going. Happy flying.

**Comment by Jeff2122** [comment]
There is so much to this I am not sure we are even talking about the same files - are they spelt bpk or bkp? 
There might be some other hidden folder in your Custom Scenery folder or something strange like that. XP will look at the folders in Custom Scenery and adjust the .ini based on what it finds. So it thinks those folders are new and adds them at the top. Something new every day.

**Reply by zippy77777** [reply]
its deleted now, it is the backup folder it creates, I might have spelt it wrong lol, sorry
